svelte-kit-scss/src/lib/components/shared/Dropdown/DropdownMenu.svelte

```diff
@@ -41,8 +49,9 @@
       font-size: 12px;
       line-height: 1;
       position: absolute;
+      z-index: 9999;
```

Comment:
I would normally advise only ever using `z-index: 1;` or `z-index: -1`, there is little need for any other values